### ISSUE TITLE
Shadow not appearing in android

### ISSUE THREAD

**Issue by Mahalakshmi269** [SteffeyDev/react-native-popover-view#135]
Describe the bug
Hello, I'm trying to render the popover as rn-modal and using popoverStyle to provide style.

Issue: 
Shadow is not working for popover in android, but shadow is appearing for popover and arrow in iOS.

Android: 
<img width="388" alt="Screenshot 2022-03-30 at 2 35 05 PM" src="https://user-images.githubusercontent.com/16955833/160798281-4b4cc1a0-7413-4a81-8774-e2d98a1ab7ee.png">

iOS: 
<img width="368" alt="Screenshot 2022-03-30 at 2 35 23 PM" src="https://user-images.githubusercontent.com/16955833/160798424-39a9eec1-d104-41ad-85e3-e56b4c8d5ed0.png">

**Device/Setup Info:**
- Device:  iOS: [iPhone 12] and Android: [Pixel 4] (Pixel_4_API_32(AVD) - 12)
- OS: [iOS 14.4]
- react-native version: [0.66.4]
- react-native-popover-view version: [5.0.0]

**Comment by Mahalakshmi269** [comment]
Used the following code: 
```
popoverStyle = {{
      backgroundColor: '#fff',
      shadowColor: '#000',
      shadowOffset: {width: 0, height: 2},
      shadowOpacity: 0.8,
      shadowRadius: 2,
      elevation: 2,
}},
arrowSize={{
      width: 32,
      height: 16,
}},
backgroundStyle={{
      backgroundColor: 'transparent',
}}
```

**Comment by BalaManikantaB** [comment]
@Mahalakshmi269 I was facing this issue too.. 
In v4.1.0 it used to show shadow on the view but after updating to latest version shadow properties are not getting applied

**Comment by SteffeyDev** [comment]
Thanks for letting me know; I just released 5.0.1 with the fix. Unfortunately, I'm not able to get the shadow working on the arrow on Android like I did on iOS (probably the way Android handles elevation and borders), but I've restored the shadow for the popover itself and raised the arrow to the same elevation, so at least the shadow won't show on top of the arrow.
